docs/BUILD.md

```diff
@@ -90,45 +91,44 @@ $ git submodule init
 $ git submodule update --recursive
 ```
 
-Install the following python packages
+###  2.1. <a name='BuildInstructions'></a>Build Instructions
+Use one of the following build options based on the requirements. **OpenVINO™ integration with TensorFlow** built using PyPI TensorFlow enable only the Python APIs, TensorFlow C++ libraries built from source is required for using the C++ APIs.
```

Comment:
Correction at below places. This is corrected version
 - <TensorFlow enables only>
 -  <required to use C++ APIs